cms/djangoapps/contentstore/views/preview.py

```diff
@@ -124,6 +125,28 @@ def user_id(self):
         return self._request.user.id
 
 
+class StudioPermissionsService(object):
```

Comment:
https://github.com/edx/edx-platform/pull/6013 is the PR that has the proposed user service in it. I don't think it's really the place we'd want to add this particular permissioning code, though (in particular, that service seems to fall in the realm of authentication (who is the user), while this one falls into the realm of authorization (what can the user do)).

I've waffled a little bit about whether calling this the `studio_user_permissions` is best or not. I'm leaning towards yes, because it makes it a bit more obvious that this is a studio-specific service. I think it would be good to have it more clearly deprecated, though, and ideally I'd love to see a concrete plan about how to design a proper authorization service to replace it.